CONTRIBUTING.rst

```diff
@@ -174,3 +174,25 @@ Then you are done.
 
 If the pull request has been opened for the ``f25-devel`` branch, then you also need to merge the ``f25-devel``
 branch back to ``master`` once you merge your pull request (see "Merging the Fedora devel branch back to the master branch" above).
+
+Pure community features
+=======================
+
+The pure community features are features which are part of the Anaconda code base but they are maintained and extended mainly by a community. These features are not
+a priority for the Anaconda project.
+
+In case there will be an issue in the pure community features, the Anaconda team will provide only a sanity checking. It is the responsibility of the
+community (maintainers of the feature) to provide fix for the issue. If the issue will have bigger impact on other parts of the Anaconda project or if it will
+block a release or another priority feature and the fix won't be provided in a reasonable time the Anaconda team have the rights to remove or disable this feature from
```

Comment:
Less words, probably same meaning, less lawyerish. Feel free to not use.
```suggestion
block a release or another priority feature and the fix won't be provided in a reasonable time the Anaconda team can remove or disable this feature from
```

Reply:
Not sure about this one. Seems to me like it has a different meaning.

Reply:
Ah, the meaning sure is different - I'd consider "have the rights" to hint at a quite formal relationship between those involved in a potential disagreement. In fact I would expect that kind of a word from a formal document, which this is not. (Is it?) That's why I went with the "can", which in my mind captures better the one-sided "I can because I am the BDFL here" reality. Because honestly, as a maintainer, you do have the right implicitly.

One further consideration is that features could be perhaps also disabled instead of removed, but I guess spending time on all details of all possibilities is not worth it. Also, the current text does go with the "worst case" outcome, so I see no problem there.

All things considered, I write all of this in the spirit of a documentation review: This is what I see and think. Make of it what you want. Therefore it's always a suggestion, not a demand/blocker :)